data/xml/2006.amta.xml

```diff
@@ -0,0 +1,511 @@
+    </paper>
+    <paper id="6">
+      <title>Can Controlled Languages Scale to the Web?</title>
+      <author><first>Jonathan</first><last>Pool</last></author>
+      <url hash="51221275">2006.amta-workshop.6</url>
+      <abstract>In a multilingual Semantic Web, authors might write in precise, expressive varieties of diverse languages. Do such controlled languages exist? Of 41 candidates, just 4 were (1) designed for multiple domains and genres and (2) documented enough for evaluation. A sample of Web statements on health and human rights revealed limited expressivity or precision in each language. The most expressive one avoided structural ambiguity but allowed semantic ambiguity that could frustrate human and machine comprehension. The possibility of a practical Web-scale controlled language remains undemonstrated but unrefuted.</abstract>
+    </paper>
+    <paper id="7">
+      <title>Standard Language at Ford Motors: A Case Study in Controlled Language Development and Deployment</title>
+      <author><first>Nestor</first><last>Rychtyckyj</last></author>
+      <url hash="f87bc23a">2006.amta-workshop.7</url>
+      <abstract>In 1990, Ford Body and Assembly Operations introduced Standard Language as a requirement for writing process sheet assembly instructions in North America. Standard Language was created to standardize the process sheet assembly instructions and introduce consistency across the entire manufacturing spectrum at Ford Motor Company. Standard Language is a Ford-specific, restricted subset of English that is used to describe the vehicle assembly process at Ford Motor Company. This language is used as the input to the Artificial Intelligence (AI) component of the Global Study Process Allocation System (GSPAS). Process sheets written in Standard Language are read by the AI system and used to generate work assembly instructions. Since its introduction in North America, Standard Language has been deployed to Ford's assembly plants in Europe, South America and Asia. In this paper, we will discuss our implementation of Standard Language at Ford Motor Company and show how a controlled language, such as Standard Language, can be used to provide a competitive advantage in business.</abstract>
+    </paper>
+  </volume>
```

Comment:
Why don't we add an entry for the Round Table, even though there's nothing associated with it? It at least records that it happened.